src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -136,7 +138,7 @@ components:
     lteRecord:
       name: LteRecord
       title: LTE Record
-      summary: Represents information recorded about an LTE eNodeB at a particular time and geographic location.
+      summary: Represents information recorded about a LTE eNodeB at a particular time and geographic location.
```

Comment:
It is "an LTE". It all depends on if it sounds like a vowel is at the start of the acronym. Here is a reference: https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html#:~:text=The%20general%20rule%20for%20indefinite,an%20HIV%20patient%20is%20correct.